src/main/kotlin/snyk/advisor/AdvisorScoreProvider.kt

```diff
@@ -178,6 +190,13 @@ class AdvisorScoreProvider(
                     """.trimIndent(),
                     e.mouseEvent
                 )
+                service<SnykAnalyticsService>().logHealthScoreIsClicked(
+                    HealthScoreIsClicked.builder()
+                        .ecosystem(packageManager.getEcosystem())
+                        .ide(HealthScoreIsClicked.Ide.JETBRAINS)
+                        .packageName(info.name)
+                        .build()
+                )
```

Comment:
Move up before `val labels` initialization? 
We can fall on `val labels` initialization or later on `SnykBalloonNotifications.showAdvisorMoreDetailsPopup` invocation and our `click` event should be registered before those possible falls.

Reply:
If we would fall on `SnykBalloonNotifications.showAdvisorMoreDetailsPopup`, should we "count" it as click. A user will not see the popup, and we'll register this as successful click 🤔 .

Reply:
clarified in JIRA ticket => resolved.